package.json

```diff
@@ -34,7 +34,6 @@
     "acorn": "^6.1.1",
     "assert-rejects": "^1.0.0",
     "aws-sdk": "^2.437.0",
-    "coveralls": "^3.0.3",
```

Comment:
If a dep was dropped, should there be a package-lock.json update too?

Reply:
Turns out it's a transitive dependency of tap, so it's still installed.